Sources/ArcGISToolkit/FloorFilter/FloorFilterViewModel.swift

```diff
@@ -0,0 +1,118 @@
+//
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import UIKit
+import ArcGIS
+
+// View Model class that contains the Data Model of the Floor Filter
+// Also contains the business logic to filter and change the map extent based on selected site/level/facility
```

Comment:
Class, property, and function declarations should be documented with three slashes:  `///`.  This is the new Apple-recommended documentation format (and again, not always represented in the existing components).  If you hit `Option-CMD-/` with the cursor on a class definition, property, or function, it will stub the necessary documentation.

```suggestion
/// View Model class that contains the Data Model of the Floor Filter.
/// Also contains the business logic to filter and change the map extent based on selected site/level/facility.
```